esp/services/esdl_svc_engine/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ set (   SRCS
         esdl_svc_engine.cpp
         esdl_store.cpp
         esdl_monitor.cpp
+        esdl_transform.cpp
```

Comment:
This is a form of transform, but because ESDL is used to do so many different types of transforms I find this file name confusing.. I would open this file looking for even the basic built in ESDL transforms.  Something like esdl_svc_custom might be better.  We can put all features related to "scripting" service behavior in there, including these custom request transforms.